docs/references/automated-image-update.md

```diff
@@ -0,0 +1,89 @@
+# Automated deployment of new container images
```

Comment:
I have also mentioned the `filter.fluxcd.io` annotation in #2298, as this annotation works for all workloads. May be worth to mention it somewhere here as 'existing'.

Reply:
Ah ok so that PR changes the annotations in the Helm docs. I'll remove it from here then.